pycqed/instrument_drivers/meta_instrument/qubit_objects/CCL_Transmon.py

```diff
@@ -1152,7 +1152,7 @@ def prepare_for_timedomain(self):
             self._prep_td_configure_VSM()
 
     def _prep_td_sources(self):
-        self.instr_spec_source.get_instr().off()
+        # self.instr_spec_source.get_instr().off()
```

Comment:
We have to keep the old functionality, in case users switch from spectroscopy to time-domain experiments. If the problem was due to the same source being used in both cases, some other solution should be found, or that use-case prevented!